packages/core/src/Queue.js

```diff
@@ -1,64 +1,106 @@
-/** @license MIT License (c) copyright 2010-2016 original author or authors */
-/** @author Brian Cavalier */
-/** @author John Hann */
+// Copied and modified from https://github.com/invertase/denque
+// MIT License
 
-// Based on https://github.com/petkaantonov/deque
+// These constants were extracted directly from denque's shift()
+// It's not clear exactly why the authors chose these particular
+// values, but given denque's stated goals, it seems likely that
+// they were chosen for speed/memory reasons.
+
+// Max value of _head at which Queue is willing to shink
+// its internal array
```

Comment:
A complete sentence should end with a period. It is also applicable to code comments. The same applies below.

Reply:
Nah. I’m good as it is. :shipit: